run_example.sh

```
@@ -9,4 +9,4 @@
 echo "## Running $className..."
 shift
 echo "## arguments $@..."
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)